src/main/java/org/opensearch/security/dlic/rest/api/AccountApiAction.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
`2019` should stay, it is either no update or `2019-2021`. Most of the time we don't update copyright year and keep initial year when copyright was established.